Compilers.md

```diff
@@ -1050,6 +1050,13 @@ Watcom C++|Open Watcom C++|`__WATCOMC__`
 |1.0|1200
 |1.7|1270
 
+## [Z88DK](https://en.wikipedia.org/wiki/Z88DK) ##
+
+Type|Macro|Format|Description
+---|---|---|---
+Identification|`__SCCZ80`<br/>`SCCZ80`||Using sccz80 compiler
+Identification|`__SDCC`  ||Using sdcc compiler
```

Comment:
While SCCZ80 doesn't look like it's standalone from Z88DK, it still might make sense to make two separate paragraphs for one for SCCZ80 in Z88DK toolchain, and SDCC, as the latter is a generic retargetable compiler.

Reply:
I never tested the standalone SDCC toolchain before so I couldn't vouch for its behaviour, but I've installed it now and it does set `__SDCC` as documented in its section above. However, I think it still belongs here with Z88DK since this is a forked version of the compiler and the `__SDCC` macro in Z88DK is only used for identification and not version, since it contains only `1`.